lib/linguist/languages.yml

```diff
@@ -1376,6 +1376,7 @@ GLSL:
   type: programming
   extensions:
   - ".glsl"
+  - ".comp"
```

Comment:
As far as I can tell, no. It seems there is a wide range of things using `.comp` extension. Should I remove this one?